frontend/src/components/Locations/LocationForm.tsx

```diff
@@ -171,38 +173,50 @@ const LocationForm = ({current, createNew, setLocation, PickupDeliveryObject, se
-                        {current.FoodType}</label>
-                  </div>
+                 <div className="flex items-center mb-4">
+                 <input 
+                       className="w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 rounded focus:ring-blue-500 dark:focus:ring-blue-600 dark:ring-offset-gray-800 focus:ring-2 dark:bg-gray-700 dark:border-gray-600 w-8 h-8 inline-block mr-2 bg-white border border-grey flex-no-shrink"
+                       id="checkbox"
+                       type="checkbox"
+                       name="foodType"
+                       value={current.FoodType}
+                       onClick={() => {
+                         active.push(current.FoodType);
+                         setActive(active);
+                         setIsClicked(true);}}
+                      />
+                   <label htmlFor="checkbox" className="flex items-center cursor-pointer text-3xl">
+                   
+                   
+                   {current.FoodType}</label>
+             </div>
               }
               {current && (
                 <div>
                   <input
-                    className='mx-3 my-5 hidden'
-                    id="radio2"
-                    type="radio"
+                    className="w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 rounded focus:ring-blue-500 dark:focus:ring-blue-600 dark:ring-offset-gray-800 focus:ring-2 dark:bg-gray-700 dark:border-gray-600  w-8 h-8 inline-block mr-2 bg-white border border-grey flex-no-shrink"
+                    id="checkbox2"
+                    type="checkbox"
                     name="foodType"
-                    onClick={() => setIsClicked(false)}
-                    checked
+                    value = {"Other"}
+                    onChange={() => {
+                      setIsClicked(!isClicked);
+                      setOtherClicked(!isOtherClicked);
+                      console.log(isOtherClicked)
+                      }}
                   />
-                  <label htmlFor="radio2" className="flex items-center cursor-pointer text-3xl">
-                        <span className="w-8 h-8 inline-block mr-2 bg-white rounded-full border border-grey flex-no-shrink"></span>
-                        {"Other"}</label>
+                  <label htmlFor="checkbox2" className="cursor-pointer text-3xl">
+                    {"Other"} </label>
+                       
                   <input className="bg-white ml-2 text-4xl w-full italic py-4 px-4 mt-2 rounded-lg shadow w-full text-left"
                     type="text"
-                    disabled={isClicked}
-                    onChange={(e) => setActive(e.target.value)}
+                    disabled= {isOtherClicked}
+                    onChange={(e) => 
+                      {
+                        active.push(e.target.value);
```

Comment:
This will push everything the user types onto the active list (i.e. it will be like [a, ap, app, appl, apple]) 
We just want to capture this value when the user hits submit. 

I'd recommend creating a state variable for `other` and only pushing its value to `active` if other is checked and the user hits submit.